projects/ngrx.io/content/guide/effects/lifecycle.md

```diff
@@ -100,18 +99,18 @@ This allows you to provide a custom behavior, such as only retrying on
 certain "retryable" errors, or with maximum number of retries.
 
 <code-example header="customise-error-handler.effects.ts">
-```ts
 import { Observable, throwError } from 'rxjs';
 import { retryWhen, mergeMap } from 'rxjs/operators';
+import { ErrorHandler, NgModule } from '@angular/core';
```

Comment:
Could you add this import to the top of the imports (so that angular gets imported first)?